assets/styles/woocommerce.css

```diff
@@ -26,3 +26,15 @@
 #optins .optional {
   display: none;
 }
+
+/* Account: Place display name fields on one line. */
+form.edit-account > .display-name .form-row-first {
+  float: left;
+}
+form.edit-account > .display-name .form-row {
+  display: inline-block;
+  width: 48%;
+}
+form.edit-account > .display-name .form-row:nth-of-type(-n+1) {
```

Comment:
Why do you write it this complex? Why don't you use the `.form-row-first` class to set the margin on the first element? For mobile views the elements should also be one per line, don't you think?

Reply:
Thanks for catching this - was an obsolete effect of a previous approach.